hack/common.sh

```diff
@@ -28,7 +28,7 @@ OUTPUT_BIN=${OUTPUT}/bin
 KUBECTL_VERSION=1.14.3
 KUBECTL_BIN=$OUTPUT_BIN/kubectl
 HELM_BIN=$OUTPUT_BIN/helm
-HELM_VERSION=3.0.1
+HELM_VERSION=3.7.2
```

Comment:
Any particular reason for helm 3.7.2? The latest is 3.9.4.

Reply:
> Any particular reason for helm 3.7.2? The latest is 3.9.4.

According to the [Helm Supported Versions](https://helm.sh/docs/topics/version_skew/#supported-version-skew), considering the compatibility we haven't tested this chart in the newer k8s version, so I choose 3.7.2. We can upgrade it later